code/modules/reagents/reagent_containers/food/mobcube.dm

```diff
@@ -136,3 +136,7 @@
 /obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/mousecube
 	name = "lab mouse cube"
 	contained_mob = /mob/living/simple_animal/mouse/balbc
+
+/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/spacecarpcube
+	name = "space carp baby cube"
```

Comment:
"baby space carp cube" seems better worded; or just space carp cube, since it makes sense to ship juveniles

Reply:
fair enough

Reply:
done